package.json

```diff
@@ -60,19 +70,20 @@
     "karma-webpack": "^1.7.0",
     "load-grunt-tasks": "^3.5.2",
     "minimist": "^1.2.0",
+    "rimraf": "^2.6.2",
     "sinon": "^1.17.4",
-    "webpack": "^1.13.1",
-    "webpack-dev-server": "^1.14.1",
     "url-search-params": "^0.6.1",
-    "typescript": "^2.0.3"
+    "webpack": "^3.10.0",
+    "webpack-dev-server": "^1.16.5"
```

Comment:
Do we need this here?

Reply:
It was just wrongly sorted in the `package.json` beforehand. The dev dependency can be removed in the Karma branch, as until then the karma config has a dependency on the Webpack dev server

Reply:
Yes to make the tests run in this PR, will get removed later on